README.md

```diff
@@ -1,38 +1,61 @@
+# Bowling Challenge
 
-Bowling Challenge
-=================
+## About
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday week
+This program calculates the score of a bowling game given the rolls. A user can input the rolls sequentially as they occur:
 
-## The Task
+```javascript
+game = new Game();
+game.bowl(5);
+game.bowl(3);
+game.score(); // => 8
+//
+```
 
-**THIS IS NOT A BOWLING GAME, IT IS A BOWLING SCORECARD. DO NOT GENERATE RANDOM ROLLS. THE USER INPUTS THE ROLLS.**
+The `game.score()` method returns the current total score, following [traditional ten-pin bowling rules](https://en.wikipedia.org/wiki/Ten-pin_bowling#Traditional_scoring). There is no need to enter `game.bowl(0)` after a strike; the game will be automatically closed when a strike occurs.
 
-Count and sum the scores of a bowling game for one player (in JavaScript).
+### Design
 
-A bowling game consists of 10 frames in which the player tries to knock down the 10 pins. In every frame the player can roll one or two times. The actual number depends on strikes and spares. The score of a frame is the number of knocked down pins plus bonuses for strikes and spares. After every frame the 10 pins are reset.
+This program follows Object-oriented design pattern, and consists of two classes: Game and Frame. Game holds an array of Frame objects. Game knows about its frames, the order of those frames and overall game score. Since Game knows the position of its frames, it is responsible for distributing their bonuses to the right frames. Frame knows about its rolls, whether it is a spare or a strike, and whether it is closed or open.
+
+## Features
+
+- Gutter game
+- One frame
+- Multiple frames
+- Spares
+- Strikes
+- Final Frame
+
+### How to run
+
+```bash
+1. Git clone https://github.com/rdupplaw/bowling-challenge.git
+2. cd bowling-challenge
+3. npm install
```

Comment:
good instructions but the numbering is unnecessary and makes it harder to copy and paste

Reply:
except that link in clone is to my github repo